frontend/src/components/Answer/AnswerRating.tsx

```diff
@@ -0,0 +1,30 @@
+import { IconButton } from "@fluentui/react";
+import { useState } from 'react';
+
+interface Props {
+    onRating?: (value: boolean | undefined) => void;
+    rating?: boolean | undefined;
+}
+
+export const AnswerRating = (props: Props) => {
+    const [rating, setRating] = useState<boolean | undefined>(props.rating);
```

Comment:
instead of undefined how about || false or remove the undefined for the rating property.

Reply:
undefined has meaning beyond false. The third state is representative of the third ui state (other than thumbs up or thumbs down).